gandalf.js

```diff
@@ -6,16 +6,17 @@
  *
```

Comment:
SCRIPT-There seems to be a typo in the function name. Did you mean `calculateAverage` instead of `calculateAverge`?